EpikCord/client/buckets.py

```diff
@@ -10,18 +10,26 @@
 
 
 class MockBucket:
+    """A mock bucket that does nothing."""
```

Comment:
Make it an abstract class or a protocol

Reply:
Alr, I'll do that soon. I was thinking about it.